src/assignments/assignments.yaml

```diff
@@ -25,25 +25,13 @@
 - yugipedia: 10276
   fake_password: 10000100
 ## Prereleases
-# Age of Overlord
-- set_abbreviation: AGOV
-  fake_password_range:
-    - 101202000
```

Comment:
TCG release is in two weeks. Do you know if we have passwords for all world-premiere cards already?

Reply:
Ah, my bad, forgot the TCG exclusives in those packs. However, I think all of their IDs have already been leaked as I just checked.

Reply:
Okay, thank you verifying :)